views/components/organisms/o-carousel.scss

```diff
@@ -0,0 +1,102 @@
+@import "../../../src/sass/variables";
+@import "../../../src/sass/common";
+@import "../atoms/a-img";
+
+.slideshow-container {
+  position: relative;
+  margin: auto;
+  box-shadow: 0 14px 6px -7px #eeeeee;
+  .prev, .next {
+    @extend .cursor-link;
+    position: absolute;
+    top: 50%;
+    width: auto;
+    @include padding(16);
+    @include margin(-22 0 0 0);
+    color: color('white');
+    font-weight: bold;
+    font-size: .9rem;
+    transition: 0.6s ease;
+    border-radius: 0 3px 3px 0;
+    user-select: none;
+    background-color: color('darkgrey');
+    opacity: .2;
+    display: none;
+    @media screen and (min-width: $tab) {
+      display: block;
+    }
+
+  }
+
+  /* Position the "next button" to the right */
+  .next {
+    right: 0;
+    border-radius: 3px 0 0 3px;
+  }
+
+
+  /* Fading animation */
+  .fade {
+    -webkit-animation-name: fade;
+    -webkit-animation-duration: 1.5s;
+    animation-name: fade;
+    animation-duration: 1.5s;
+  }
+
+  @-webkit-keyframes fade {
+    from {opacity: .4} 
+    to {opacity: 1}
+  }
+
+  @keyframes fade {
+    from {opacity: .4} 
+    to {opacity: 1}
+  }
+  .tab{
+    display:block;  
+    @media screen and (max-width: $mob) {
+        display:none;  
+    }
+    @include padding(0 0 10 0);
+    @extend .text-center;
+    .dot {
+      @extend .cursor-link;
+      height: 5px;
+      width: 5px;
+      margin: 0 10px;
+      background-color: #bbb;
+      border-radius: 50%;
+      display: inline-block;
+      transition: background-color 0.6s ease;
+    }
+
+    .active, .dot:hover {
+      background-color: #717171;
+    }
+  }
+
+
+}
+
+  /* The dots/bullets/indicators */
+  .mob{
```

Comment:
The class names need to be context specific

Reply:
I have changed the class names to meaningful ones.